docs/UnityGroceriesRealWorld.md

```diff
@@ -0,0 +1,44 @@
+# UnityGroceries-RealWorld Dataset
+
+As part of this project, we collected a real world dataset containing 1267 images of the 63 corresponding grocery items. This dataset is primary used to compare performance of model trained on the Synthetic counterparts. This dataset can be downloaded [here](https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip).
+
+## dataset collection
+
+To take the photos for the UnityGroceries-RealWorld dataset, we devised a strategy to vary object selection, placement, lighting, and background. After a few test shoots, we realized that major barriers to randomness of placement and object selection were the actual humans doing the selecting and placing. Hence, planning on how to randomize while taking photos became highly important, and something we did before taking photos.
```

Comment:
If you want to use the proper name for the dataset, introduce it at the beginning of the article. I think it sounds fine without "UnityGroceries-RealWorld".

Reply:
Agree. I removed the dataset name here. We don't need to repeat the name here.